assets/dev/scss/frontend/widgets/link-in-bio.scss

```diff
@@ -0,0 +1,93 @@
+.elementor-widget-link-in-bio {
+	--align-self: center;
+	--e-link-in-bio-gutter: 2em;
+
+	.elementor-widget-container {
+		border: 1px solid blue;
+		margin-inline: auto;
+		max-width: 100%;
+		padding: var(--e-link-in-bio-gutter);
+		padding-block-start: 0;
+		width: 100%;
+	}
+}
+
+.e-link-in-bio {
+
+	&__content-container {
+		display: flex;
+		flex-direction: column;
+		gap: 1em;
+		margin-inline: auto;
+		text-align: center;
+
+		* {
+			word-wrap: break-word;
+		}
+	}
+
+	&__identity--profile {
+		padding-block-start: var(--e-link-in-bio-gutter);
+
+		img.e-link-in-bio__identity-image {
+			aspect-ratio: 1;
+			border: 1px solid blue;
+			border-radius: 50%;
+			display: block;
+			object-fit: cover;
+			margin-inline: auto;
+			max-width: 100%;
+			width: 150px;
+		}
+	}
+
+	&__identity--cover {
+
+		img.e-link-in-bio__identity-image {
+			display: block;
```

Comment:
I am not sure if we are all still using `display: block`;
I would prefer:
```
display: flex;
flex-direction: column;
```

Reply:
@hein-obox the use of display: block here is purely on the img element itself rather than a wrapper that would benefit from flex model. Give that, is this still an issue? Thanks